website/content/zh/wgs/platforms/whitepaper/index.md

```diff
@@ -2,504 +2,199 @@
-  </tr>
-  <tr>
-    <td>Automation for building and testing products</td>
-    <td>Automate build and test of digital products and services.</td>
-    <td>Tekton, Jenkins, Buildpacks, ko, Carvel</td>
-  </tr>
-  <tr>
-    <td>Automation for delivering and verifying services</td>
-    <td>Automate and observe delivery of services.</td>
-    <td>Argo, Flux, Keptn, Flagger, OpenFeature</td>
-  </tr>
-  <tr>
-    <td>Development environments</td>
-    <td>Enable research and development of applications and systems.</td>
-    <td>Devfile, Nocalhost, Telepresence, DevSpace</td>
-  </tr>
-  <tr>
-    <td>Application observability</td>
-    <td>Instrument applications, gather and analyze telemetry and publish info to stakeholders.</td>
-    <td>OpenTelemetry, Jaeger, Prometheus, Thanos, Fluentd, Grafana, OpenCost</td>
-  </tr>
-  <tr>
-    <td>Infrastructure services</td>
-    <td>Run application code, connect application components and persist data for applications</td>
-    <td>Kubernetes, Kubevirt, Knative, WasmEdge, KEDA<br />CNI, Istio, Cilium, Envoy, Linkerd, CoreDNS<br />Rook, Longhorn, Etcd</td>
-  </tr>
-  <tr>
-    <td>Data services</td>
-    <td>Persist structured data for applications</td>
-    <td>TiKV, Vitess, SchemaHero</td>
-  </tr>
-  <tr>
-    <td>Messaging and event services</td>
-    <td>Enable applications to communicate with each other asynchronously</td>
-    <td>Strimzi, NATS, gRPC, Knative, Dapr</td>
-  </tr>
-  <tr>
-    <td>Identity and secret services</td>
-    <td>Ensure workloads have locators and secrets to use resources and capabilities. Enable services to identify themselves to other services</td>
-    <td>Dex, External Secrets, SPIFFE/SPIRE, Teller, cert-manager</td>
-  </tr>
-  <tr>
-    <td>Security services</td>
-    <td>Observe runtime behavior and report/remediate anomalies. Verify builds and artifacts don't contain vulnerabilities. Constrain activities on the platform per enterprise requirements; notify and/or remediate aberrations</td>
-    <td>Falco, In-toto, KubeArmor, OPA, Kyverno, Cloud Custodian</td>
-  </tr>
-  <tr>
-    <td>Artifact storage </td>
-    <td>Store, publish and secure built artifacts for use in production. Cache and analyze third-party artifacts. Store source code.</td>
-    <td>ArtifactHub, Harbor, Distribution, Porter</td>
-  </tr>
-</table>
-
-<!-- ## Footnotes -->
-
-[1]: https://www.atlassian.com/devops/frameworks/team-topologies
-[2]: https://martinfowler.com/articles/talk-about-platforms.html
-[3]: https://teamtopologies.com/key-concepts-content/what-is-a-thinnest-viable-platform-tvp
-[4]: https://queue.acm.org/detail.cfm?id=3454124
-[5]: https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling
+正如我们所描述的，用于云原生计算的平台提供和组合来自许多支持提供者的功能和服务。这些提供者可以是同一企业内的其他团队，也可以是云服务提供商等第三方。简而言之，平台从基础 *能力提供者* 到平台用户（如应用程序开发人员）的中间桥梁；在此过程中，实现和执行所需的安全性、性能、成本治理和一致的体验。以下图形说明了产品、平台和能力提供者之间的关系。
+
+![产品、平台和能力提供者之间的关系](platform_components.png)
```

Comment:
Hi @rootsongjc  I pushed the Chinese version of this picture, so need you to update this URI by platforms-def.drawio.zh.png . 
You can see PR #509 for more detailed info.